### PR TITLE
Added exploitDB resource

### LINKED ISSUE #2075: [Add] Cybersecurity resources
### Site URL

https://www.exploit-db.com/

### Description of the above provided site

This site is a complete database for publicly available exploits for the vulnerabilities that are getting disclosed day to day. Along with this, this site also contains a huge database of Google Dorks.

### Category

Cyber Security

### Sub Category

security resources

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am participating in Hacktoberfest 2023

### PR DESCRIPTION
**Pull request by dellucifer**
## Fixes Issue
Closes #2075

## Changes proposed

- I have updated the repo with an awesome and needful resource which is a whole database of publicly available exploits.
- This database is from offensive security.

## Screenshots

Screenshot 2023-10-24 123236

## Note to reviewers

According to my experience, this is a mandatory resource for cybersecurity geeks. It helps them to know more about the technical aspects of the vulnerabilities by analysing the exploits.

### CODE REVIEW

**Comment by dellucifer** [comment]
@rupali-codes @Anmol-Baranwal @CBID2, kindly review my PR. Let me know if there are any changes.

File: database/cybersecurity/offensive_security.json [hunk 1]
```diff
@@ -44,5 +44,12 @@
     "url": "https://zsecurity.org/",
     "category": "cyber-security",
     "subcategory": "offensive-security"
+  },
+  {
+    "name": "ExploitDB",
+    "description": "ExploitDB is a full-fledged database of publicly available exploits for the disclosed vulnerabilities till date.",
```
Comment (CBID2) [inline database/cybersecurity/offensive_security.json:50]:
```suggestion
    "description": "ExploitDB is a full-fledged database of publicly available exploits for disclosed vulnerabilities.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
You're close @dellucifer. Just need to make a few more edits

**Comment by dellucifer** [comment]
Suggested edits are done.

**Review by CBID2 — APPROVED** [review]
Looks great! :) @Anmol-Baranwal & @rupali-codes, you're up! :)

**Comment by Anmol-Baranwal** [comment]
@dellucifer 
Following the PR template is a must. It includes special words like 'fixes' or 'resolves' that connect your issue to the PR, as explained in the official docs.

If you don't link the issue correctly, even after the PR is merged, the issue might stay open, causing huge problems.

> You can see here.
image

I'm doing it for you this time. Please ensure it next time.

Just a heads-up to help you link issue properly next time. Have a great day!

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

**Review by rupali-codes — APPROVED** [review]
lgtm

**Pull request merged**